lagom-eventstore-example/README.md

```diff
@@ -0,0 +1,99 @@
+# Lagom integration with IBM Project EventStore
+
+[IBM Project EventStore](https://www.ibm.com/us-en/marketplace/project-eventstore) is an in-memory database designed for massive structured data volumes and real-time analytics built on Apache SPARK and Apache Parquet Data Format. 
+
+This project demonstrates a simple Lagom service that includes a [Read-Side](https://www.lagomframework.com/documentation/current/java/ReadSide.html) processor which stores events of the [Persistent Entities](https://www.lagomframework.com/documentation/current/java/ReadSide.html) into Project EventStore for further analysis.
+
+The Lagom service exposes a single HTTP endpoint to simulate events.
+
+## Prerequisites
+
+To build and run this example, you need:
+
+- [git](https://git-scm.com/)
+- [Java SE 8 JDK](http://www.oracle.com/technetwork/java/javase/overview/index.html)
+- [Maven 3.2.1+](https://maven.apache.org/) to build and run the Lagom project (3.5.0 recommended)
+- IBM Project EventStore [downloaded](https://github.com/IBMProjectEventStore/EventStore-DeveloperPreview/releases/tag/1.1.1), [installed](https://www.ibm.com/support/knowledgecenter/SSGNPV/eventstore/desktop/welcome.html) and running on your local machine. (This process requires 12+Gb of downloads)
+
+Once you have an IBM Project EventStore running in your machine, the main steps to run this example are:
+
+1.  [Download and set up the Lagom service](#download-and-set-up-the-lagom-service)
+2.  [Start the Lagom sample application](#start-the-lagom-sample-application)
+3.  [Generate some traffic on the Lagom service](#generate-some-traffic-on-the-lagom-service)
+4.  [Analyse the traffic using IBM Project EventStore Notebooks](#analyse-the-traffic-using-ibm-project-eventstore-notebooks)
+
+## Download and set up the Lagom service
+
+Follow these steps to get a local copy of this project and configure it with the Message Hub credentials you saved in the previous step. You can supply the credentials in a configuration file or as environment variables.
+
+1.  Open a command line shell and clone this repository:
+    ```
+    git clone https://github.com/typesafehub/lagom-ibm-integration-examples.git
```

Comment:
this is pointing to the old repo, which makes me wondering if the other README files are also doing the same.

Reply:
Good call, we'll probably have to review all files.

Reply:
Yes, I noticed this today, too. I'll fix the existing projects in a separate pull request.

Reply:
https://github.com/lagom/ibm-integration-examples/pull/3